jupyter_server_proxy/handlers.py

```diff
@@ -457,6 +460,32 @@ def get_timeout(self):
                 return False
 
     async def ensure_process(self):
+        """
+        Start the process
+        """
+        # The default asyncio event loop implementation on Windows does not 
+        # support subprocesses. Subprocesses are available for Windows if a 
+        # ProactorEventLoop is used. 
+        # Ref: https://docs.python.org/3/library/asyncio-subprocess.html
+        # We use a workaround for windows by starting a new ProactorEventLoop
+        # in a thread. This is necessary because we don't want it to interfere with
+        # the event loop for tornado web server (which does not support ProactorEventLoop).
+        if sys.platform == "win32":
+            def start_thread():
+                new_loop = asyncio.ProactorEventLoop()
+                # patch add_signal_handler for simpervisor
+                def add_signal_handler(signum, callback, *args):
```

Comment:
Simpervisor needs these to watch for child processes dying so it can restart them, which is the core purpose of simpervisor. I'm guessing that doesn't work on Windows? If so, we might need an entirely different implementation in simpervisor for windows to work...

Reply:
The `ProactorEventLoop` on windows has no implementation for `add_signal_handler`.